cmd/ketch/app_deploy_test.go

```diff
@@ -150,6 +151,7 @@ kubernetes:
 web: python app.py
 worker: python app.py
 `
+	packBuildMetadata string = "{\"bom\":null,\"buildpacks\":[{\"id\":\"heroku/python\",\"version\":\"0.3.1\"},{\"id\":\"heroku/procfile\",\"version\":\"0.6.2\"}],\"launcher\":{\"version\":\"0.11.3\",\"source\":{\"git\":{\"repository\":\"github.com/buildpacks/lifecycle\",\"commit\":\"aa4bbac\"}}},\"processes\":[{\"type\":\"web\",\"command\":\"python app.py\",\"args\":null,\"direct\":false,\"buildpackID\":\"heroku/procfile\"},{\"type\":\"worker\",\"command\":\"python app.py\",\"args\":null,\"direct\":false,\"buildpackID\":\"heroku/procfile\"},{\"type\":\"worker1\",\"command\":\"python app.py\",\"args\":null,\"direct\":false,\"buildpackID\":\"heroku/procfile\"}]}"
 )
 
 func TestNewCommand(t *testing.T) {
```

Comment:
I think there is an upcoming issue with these unit tests: they are not unit tests. There are tests that cover few cases for the top-level function and no tests that have good coverage for small functions.

Reply:
Do you mean testing the functions in deploy.go individually?